src/drivers/imu/adis16477/ADIS16477.cpp

```diff
@@ -260,10 +166,10 @@ ADIS16477::probe()
 
 	// read product id (5 attempts)
 	for (int i = 0; i < 5; i++) {
-		_product = read_reg16(PROD_ID);
+		uint16_t product = read_reg16(PROD_ID);
```

Comment:
I'd love to see var names like this be absolutely "no-brainer"... there is no need to shorten product_id in this age. :)

Reply:
Makes sense, done.